From c12e612648435c40a704150b73cd583485257a14 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Sat, 30 Apr 2022 00:45:49 +0200
Subject: [PATCH] Add additional permissions for

 org.webosports.notifications

Solves:

Apr 29 17:42:30 qemux86-64 ls-hubd[236]: [] [pmlog] ls-hubd LSHUB_NO_NAME_PERMS {"APP_ID":"org.webosports.notifications","ROLE_ID":"/usr/bin/surface-manager","EXE":"/usr/bin/surface-manager"} Executable: "/usr/bin/surface-manager" (cmdline: /usr/bin/surface-manager -platform eglfs -plugin VBoxTouch:/dev/input/event3:/dev/input/event4:/dev/input/event5) does not have permission to register name: "org.webosports.notifications"

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

---
 base/sysbus/com.webos.surfacemanager.role.json.in | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/base/sysbus/com.webos.surfacemanager.role.json.in b/base/sysbus/com.webos.surfacemanager.role.json.in
index 739adac..462991f 100644
--- a/base/sysbus/com.webos.surfacemanager.role.json.in
+++ b/base/sysbus/com.webos.surfacemanager.role.json.in
@@ -3,7 +3,8 @@
     \"type\": \"privileged\",
     \"trustLevel\": \"oem\",
     \"allowedNames\": [
-        \"com.webos.surfacemanager*\"
+        \"com.webos.surfacemanager*\", 
+        \"org.webosports.notifications\"
     ],
     \"permissions\": [
         {
@@ -14,6 +15,15 @@
                 \"com.webos.*\",
                 \"org.webosports.*\"
             ]
+        },
+        {
+            \"service\": \"org.webosports.notifications\",
+            \"outbound\": [
+                \"com.lge.*\",
+                \"com.palm.*\",
+                \"com.webos.*\",
+                \"org.webosports.*\"
+            ]
         }
     ]
 }
